code/modules/food/recipes_microwave.dm

```diff
@@ -1170,8 +1170,41 @@ I said no!
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/spicy_clams
 
-
 /datum/recipe/earthenroot_soup
 	fruit = list("earthenroot" = 2)
 	reagents = list("water" = 10, "spacespice" = 1, "sodiumchloride" = 1)
-	result = /obj/item/weapon/reagent_containers/food/snacks/earthenroot_soup
\ No newline at end of file
+	result = /obj/item/weapon/reagent_containers/food/snacks/earthenroot_soup
+
+/datum/recipe/onionrings
+	fruit = list("onion" = 1)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/doughslice
```

Comment:
I think this would be better if it was a batter/deep fry recipe, but it doesn't really matter.